pkgs/development/libraries/science/biology/elastix/default.nix

```diff
@@ -2,12 +2,12 @@
 
 stdenv.mkDerivation rec {
   pname    = "elastix";
-  pversion = "4.9.0";
+  pversion = "5.0.0";
   name  = "${pname}-${pversion}";
```

Comment:
Just
```nix
    pname = "elastix";
    version = "5.0.0";
```
is enough (`name` will be defined automatically)

Reply:
Updated.